tests/www/test_views.py

```diff
@@ -28,7 +28,7 @@
 import sys
 import json
 
-from urllib.parse import quote_plus
+from six.moves.urllib.parse import quote_plus
```

Comment:
NIT: Are you sure we want to use six here ? It will be easier to cherry-pick but maybe we should switch to python3 urrllib here ?

Reply:
This is for the classic UI - so py2 needed (and this UI is deleted in master)